app.ts

```diff
@@ -0,0 +1,287 @@
+    <button value="next" class="next-records-btn">Next</button>
+    <button value="last" class="last-page-btn">Last Page</button>  
+    <button onclick="recordSelection()" id="confirmation-btn">Get Record</button>
+  </div>
+  <div class="current-page-container">
+    <p class=current-page></p>
+  </div>
+  `;
+};
+
+function recordSelection() {
+  let selectionArea: any = document.querySelector("#record-navigation-container");
+  let singleRecordSelection = `
+    <button id="return-btn">Return</button>
+      <div id="user-input-data">
+        <div class="navigation-input-area-id" id="id">
+          <label class="record-labels" for="record-id"
+            >Enter record ID :
+          </label>
+          <input
+            type="text"
+            min="0"
+            minlength="1"
+            maxlength="6"
+            name="record-id"
+            id="record-id"
+            class="navigation-input"
+            value="0"
+          />
+        </div>
+        <p class="amount-of-records"></p>
+      </div>
+    <button id="get-record-btn">See Record</button>
+    `;
+
+  selectionArea.innerHTML = singleRecordSelection;
+
+  let returnBtn: any = document.querySelector("#return-btn");
+  let recordIdValue: any = document.querySelector("#record-id.navigation-input");
+  let numberOfRows = Math.floor(window.innerHeight / 50);
+
+  returnBtn.addEventListener("click", () => {
+    window.location.reload();
+  });
+
+  let getSingleRecord: any = document.querySelector("#get-record-btn");
+
+  getSingleRecord.addEventListener("click", () => {
+    fromNumber = Number(recordIdValue.value);
+    let toNumber = fromNumber + numberOfRows;
+    let finalRecord = recordNumberTotal - 1;
+
+    if (toNumber > finalRecord) {
+      toNumber = finalRecord;
+      fromNumber = toNumber - numberOfRows;
+    }
+
+    let check = ["undefined", "string", ""];
+
+    try {
+      if (check.includes(typeof fromNumber) || fromNumber < 0) {
+        alert("Does not exists");
+        recordIdValue.value = "0";
+      } else if (typeof fromNumber === "number" && fromNumber >= 0) {
+        getRecords(fromNumber, toNumber);
+      }
+    } catch (error) {
+      alert(error);
+    }
+  });
+}
+
+function createHeadingGrid(headings: any) {
+  let headingsData: any = `<h1 class="column-heading">${headings}</h1>`;
+  headingColumns.innerHTML += headingsData;
+}
+
+let recordCount = () => {
+  fetch("http://localhost:2050/recordCount", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((response) => response.text())
+    .then((data) => {
+      recordNumberTotal = JSON.parse(data);
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+let headingRowCreation = () => {
+  fetch("http://localhost:2050/columns", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((response) => response.text())
+    .then((data) => {
```

Comment:
Please remove the `()` around the parameters or specify the types, as the compiler currently thinks they are of type `any`.